dpm/client/__init__.py

```diff
@@ -4,14 +4,41 @@
 from .do_validate import validate
 
 import os
+
 import datapackage
+import requests
+import six
+
 
 class DpmException(Exception):
     pass
 
+class NetworkError(Exception):
```

Comment:
Personally, i think we could just have a single DpmException and just change the message. We don't need to distinguish types of error for us at the class level.

And if we *are* going to have different types we should inherit from DpmException.